milestone-three

RoyNyaga opened the pull request:
In this branch, we added a wide and color props to the button component then styled the calculator.

Oghenebrume50 commented on src/styles/App.css:
```diff
@@ -0,0 +1,3 @@
+#app-wrapper-div {
```
This should use flex box ![see this](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/useFlex.png)
Also the display and buttonpanel components, please make changes accordingly

RoyNyaga replied:
Ok will do that

Oghenebrume50 commented on src/components/Button.js:
```diff
@@ -4,13 +4,20 @@ import PropTypes from 'prop-types'
 
 const Button = (props) => {
+  // var allowRegister = age > 18
+  console.log(props.wide)
```
Please remove commented codes and delete console.log,  these things are only for debugging they should not make it to production.

Oghenebrume50 commented on src/components/ButtonPanel.js:
```diff
@@ -1,38 +1,40 @@
-        <Button name='-' classNaming='generalBtn' />
+        <Button name='4' color='##ffffff' wide={false} />
+        <Button name='5' color='##ffffff' wide={false} />
+        <Button name='6' color='##ffffff' wide={false} />
```
When a property is boolean, you do not need to explicitly pass it false, not passing it at all would do the trick, so here the wide prop can be removed and for all other occurrences were it is set to false

Oghenebrume50 commented on src/styles/Button.css:
```diff
@@ -1,9 +1,7 @@
-.generalBtn {
-	width: 100px;
-	height: 60px;
+.normal {
```
The button component should have borders ![see this](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/ButtonsBoarder.png)

RoyNyaga commented:
Used display flex property on app buttonPanel and button component. Removed commented codes and console logs. Removed wide prop on buttons which shouldn't display a size twice as normal so as to make it resolve to false. Added borders to the button element.

RoyNyaga commented:
yuuupppppiiiiiiii will do just that before merging.

On Fri, Feb 7, 2020 at 12:28 PM SHEY Louis Chia <notifications@github.com>
wrote:

> *@shloch* approved this pull request.
> hello @RoyNyaga <https://github.com/RoyNyaga>
>
> Great job
>
> [image: Screen Shot 2020-02-07 at 12 27 17 PM]
> <https://user-images.githubusercontent.com/40712459/74025867-3375a180-49a5-11ea-915a-cefe320c6453.png>
>
> Please enlarge font size before merging. You can see it's small. Not in
> requirement but that will be good. 😊 😊
> YOU GET A THUMBS UP FROM ME 👏👏👏👏👏
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/RoyNyaga/microverse-calculator-js/pull/3?email_source=notifications&email_token=ALZGFHPAF4HXNFDLETURYJDRBVAWVA5CNFSM4KRES25KYY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCUVFF3A#pullrequestreview-355095276>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALZGFHNRG6DECKI75PKHFVLRBVAWVANCNFSM4KRES25A>
> .
>
